update snyk documentation (#459)
guardian/security-hq: hq/markdown/snyk.md

This may not work if you are already signed into another Snyk account, so you may need to logout first.

We have an enterprise group called 'The Guardian'. Everyone will get added to the organisation `guardian-people`
(under our enterprise group) when they first log in. Once team members have signed in, they should be added

> to other organisations for each team. You will not be able to see teams you are not a member of.

Reviewer: ```suggestion
to other organisations corresponding to teams they are a part of. You will not be able to see organisations you are not a member of.
```

Developers can be added to a team by a group admin (a few individuals or the InfoSec team) or an admin of the desired
organisation. You will probably also want to upgrade team members from collaborators to admins (in the members
section of the dashboard).

## Integrating Snyk with your project(s)

By far the most effective way to integrate Snyk is through GitHub Actions. Setting up this Action will send a list of
your project's dependencies to snyk.io whenever a new push to `main` takes place. You can find how to do this in the
[snyk actions repo](https://github.com/guardian/.github/tree/main/.github/workflows). Snyk will take this list and compare
it to a list of vulnerable dependencies daily. If any new ones are found, team members will be notified by email, and it
will show up on the UI as a vulnerable project. If a patch is available, it will indicate this. A failed action does not
indicate that your project is vulnerable, it just means the action was unable to complete. This may be because it can't
communicate with snyk's servers, or perhaps a required plugin has been removed.

> This snyk action can produce feedback a little more slowly than other methods, so for an extra layer of security, we

Reviewer: I don't think it creates a project in snyk, so we should be safe. Here's an example of one on security HQ: https://github.com/guardian/security-hq/blob/main/.github/workflows/snyk-pr.yml.

I think the PR check should probably be recommended but not required. We want teams to catch vulnerabilities as early as possible and to ease their workflow, so PR checks (and IDE plugins) seem like good recommendations.
